### DISCUSSION TITLE
数字 0 与 大写字母 O 经常识别错误

### DISCUSSION THREAD

**Discussion by dengmingD** [PaddlePaddle/PaddleOCR#12916] [Q&A]
![001_12](https://github.com/PaddlePaddle/PaddleOCR/assets/72127773/a62216f9-4968-47b8-aabb-f62af929ad68)
 数字 0 与 大写字母 O 经常识别错误

rec_v3,rec_v4,rec_server_v4,svtr都识别错误

**Comment by thsno02** [comment]
如果你的字体固定的话，fine-tuning 会改善。

**Comment by XujiRe** [comment]
你好，请问后面解决了吗？我现在也遇到这个问题0091识别成0O91或者OO91了
